tests/e2e/support/commands.js

```diff
@@ -164,19 +153,17 @@ Cypress.Commands.add('signupOpponent', (username, password) => {
     );
   });
 });
-
-Cypress.Commands.add('signupPlayer', (username, password) => {
+Cypress.Commands.add('signupPlayer', (player) => {
```

Comment:
If we destructure above then we would do it here too
```suggestion
Cypress.Commands.add('signupPlayer', ({ username, password }) => {
```